.github/workflows/release.yml

```
@@ -62,34 +72,44 @@ jobs:
           GA4_SECRET: ${{ secrets.GA4_SECRET }}
           VITE_IS_BETA: ${{ github.event.release.prerelease == true }}
 
+      - name: 🔔 Create deployment notification
+        uses: agrc/service-now-worknote-action@v1
+        with:
+          repo-token: ${{ github.token }}
+          username: ${{ secrets.SN_USERNAME }}
+          password: ${{ secrets.SN_PASSWORD }}
+          instance-name: ${{ secrets.SN_INSTANCE }}
+          table-name: ${{ secrets.SN_TABLE }}
+          system-id: ${{ secrets.SN_SYS_ID }}
+
   windows:
```

```suggestion
  deploy-windows:
```
Just for consistency with the deploy-macos name.